Formula/l/lzsa.rb

```diff
@@ -0,0 +1,19 @@
+class Lzsa < Formula
+  desc "Lossless packer that is optimized for fast decompression on 8-bit micros"
+  homepage "https://github.com/emmanuel-marty/lzsa"
+  url "https://github.com/emmanuel-marty/lzsa/archive/refs/tags/1.4.1.tar.gz"
+  sha256 "c65ca1e6a43696f4ca5edc2c98229fba1044806bd21bc2a8ce4b867dc9cfc45c"
+  license all_of: ["Zlib", "CC0-1.0"]
+
+  def install
+    system "make"
+    bin.install "lzsa"
```

Comment:
no `make install`?

Reply:
The Makefile doesn't have an `install` target; it just builds the binary in-place. This is a dependency for another formula, and not my code; I can recommend changes but don't want to be blocked.